bin/mail/2017-05-17-end-of-year/send-final-real-mails

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+../send-to-mailing-list 2017-05-17-endofyear-k5pd ./email_1_csf_pd.csv
+../send-to-mailing-list 2017-05-17-endofyear-k5nopd ./email_2_csf_no_pd-remainder.csv
```

Comment:
Do you want to send the remainder from the A/B tests in this initial barrage? I think you want to send this separately sometime later, after determining the winner.